src/commands/generate/index.ts

```diff
@@ -57,14 +72,30 @@ export default class Hello extends Command {
               name: 'Complete infrastructure (VPC + RDS + LOG + S3 + FARGATE + LOG + Security groups + ALB)',
             },
           ],
-        }
-      ]
+        },
+      ];
+
+      const infrastructureType = await inquirer.prompt(questions);
 
-      const infrastructureType = await inquirer.prompt(questions)
+      const options: GenerateOption = {
+        projectName: args.projectName,
+        platform: platformChoice.platform,
+        infrastructureType: infrastructureType.infrastructureType,
+      };
 
-      console.log(infrastructureType)
-      
-      Advanced.run()
+      console.log(options);
+
+      switch (options.infrastructureType) {
+        case 'advance':
+          Advanced.run(options);
+          break;
+        case 'basic':
+        default:
+          console.log('Not implemented this type yet');
```

Comment:
```suggestion
          console.log('This type has not been implemented.');
```

Reply:
Fixed in https://github.com/nimblehq/infrastructure-templates/pull/27/commits/3d71b5d98ab51ef06ee7dd1d9fb47b3b474d6026